tests/backstop_data/engine_scripts/puppet/mosaico/mailing/step-2.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = async (engine, scenario, viewport) => {
+  await require('./step-1.js') (engine, scenario, viewport);
+  // Wait for Angular to attach events
+  await engine.waitFor(200);
```

Comment:
same as above, we should avoid doing this. What if, on one dev's local environment, angular is slower?